pages/glossary/overlap.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Overlap
+key: overlap
+unambiguous: true
+objective: true
+input_aspects:
+  - CSS styling
+  - DOM tree
+---
+
+[Bounding boxes][] A and B _overlap_ if at least one point inside box A has the same [left][left coordinate] and [top coordinates][top coordinate] of one point inside box B.
```

Comment:
Points don't have a left / to pcoordinate. I would suggest you define this as any corner of element A is between the left and right, and top and bottom edges of element B, or visa versa.

Reply:
Correct. Thanks for the suggestion.